packages/circuits/test/proveSubsidyKey.test.ts

```diff
@@ -0,0 +1,320 @@
+import * as path from 'path'
+import { expect } from 'chai'
+import { genRandomSalt, ZkIdentity, hashOne } from '@unirep/crypto'
+import * as crypto from '@unirep/crypto'
+import * as circom from 'circom'
+import * as snarkjs from 'snarkjs'
+import {
+    USER_STATE_TREE_DEPTH,
+    GLOBAL_STATE_TREE_DEPTH,
+    NUM_EPOCH_KEY_NONCE_PER_EPOCH,
+    EPOCH_TREE_DEPTH,
+} from '@unirep/circuits'
+import { Reputation } from '../src/Reputation'
+
+const circuitName = 'proveSubsidyKey'
+const circuitPath = path.join(
+    __dirname,
+    `../circuits/test/${circuitName}_test.circom`
+)
+
+const genEpochKey = (
```

Comment:
it can be imported from `@unirep/core` as well

Reply:
d9b016e30202f701fb47e0e7db4efdfa0e79bb9f